jsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
How do you want to handle comments for .json files? Is there a step in the preprocessor that strips them?